pkg/diagnostics/v2/client_test.go

```diff
@@ -0,0 +1,55 @@
+package v2
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+	"github.com/dcos/dcos-diagnostics/api/rest"
```

Comment:
I'm not sure we should add this strict coupling here but rather only depend on HTTP and JSON as an API.